kf_lib_data_ingest/app/cli.py

```diff
@@ -197,10 +197,10 @@ def cli():
     "the target service. Overrides the resume_from setting.",
 )
 @click.option(
-    "--no_warehouse",
+    "--warehouse",
     default=False,
```

Comment:
This means the `warehouse` kwarg is set to `False`  by default right? I can't remember click

Reply:
It _should_ mean that :D

Reply:
I think a boolean toggle must always default to false, so it probably doesn't even need to be there. Like if it defaulted to true, how would you set it false right?